DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -151,9 +152,25 @@ public override void Initialize()
             tunnelSelectionWindow.CenterOnParent();
             tunnelSelectionWindow.Disable();
 
+            btnChangeTunnel = new XNAClientButton(WindowManager);
+            btnChangeTunnel.Name = "btnChangeTunnel";
+            btnChangeTunnel.ClientRectangle = new Rectangle(btnLeaveGame.Right - btnLeaveGame.Width - 145,
+                btnLeaveGame.Y, 133, 23);
+            btnChangeTunnel.Text = "Change Tunnel";
+            btnChangeTunnel.LeftClick += BtnChangeTunnel_LeftClick;
+
+            btnChangeTunnel.Enabled = false;
+            btnChangeTunnel.Visible = false;
```

Comment:
Isn't making it disabled by default like this rather pointless since `SetUp` is always called when the game room is opened?